src/test/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
add application.properties to src/main/resources and provide spring.datasource properties to connect to the DB 
Also, don't forget 
```
spring.jpa.hibernate.ddl-auto=create-drop
spring.jpa.show-sql=true
```